### PR TITLE
added zone_id requirements to README file

### PR DESCRIPTION
**Pull request by cerberillo**
This patch adds ZONE ID as a requirement to the README file so we specify it as a required field as it is from now on

### CODE REVIEW

**Comment by cerberillo** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;ISq2KCDl&#x2F;1284-create-new-interface-method-with-zoneid-requirement"> Create new interface method with zoneID requirement</a>

**Comment by cerberillo** [comment]
I need a new review @RagnarR @anshumansworld

File: library.demo/src/main/java/net/pubnative/library/demo/utils/Settings.java [hunk 1]
```diff
@@ -20,4 +22,18 @@ public static String getAppToken() {
     public static void setAppToken(String appToken) {
         mAppToken = appToken;
     }
+
+
+
```
Comment (anshumansworld) [inline library.demo/src/main/java/net/pubnative/library/demo/utils/Settings.java]: extra lines :)

File: library.demo/src/main/res/layout/activity_settings.xml [hunk 2]
```diff
@@ -11,40 +11,69 @@
...
             android:focusableInTouchMode="true"
             android:textColor="#2e2e2e"/>
 
+
         <TextView
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:text="APP TOKEN"
-            android:id="@+id/textView2"
+            android:text="ZONE ID"
```
Comment (anshumansworld) [inline library.demo/src/main/res/layout/activity_settings.xml]: we can use it from strings.xml

File: library.demo/src/main/res/layout/activity_settings.xml [hunk 3]
```diff
@@ -11,40 +11,69 @@
...
             android:padding="5dp"
             android:textSize="20dp"
-            android:textStyle="bold"
-            android:layout_alignParentTop="true"/>
+            android:textStyle="bold"/>
+
+        <EditText
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:text="ZONE ID"
```
Comment (anshumansworld) [inline library.demo/src/main/res/layout/activity_settings.xml]: same

File: library/src/main/java/net/pubnative/library/request/PubnativeRequest.java [hunk 4]
```diff
@@ -55,6 +55,9 @@
 
     private static String TAG = PubnativeRequest.class.getSimpleName();
 
+
```
Comment (anshumansworld) [inline library/src/main/java/net/pubnative/library/request/PubnativeRequest.java]: we can remove extra line

**Comment by cerberillo** [comment]
@anshumansworld when reviewing, there is an option "request changes" that blocks the PR from merge until it's approved, next comments let's use that option instead the "comment" option one :)

**Pull request merged**